docker-entrypoint.sh

```diff
@@ -36,22 +36,60 @@ if [ "$1" = "help" ]; then
     exit 0
 elif [ "$1" = "jobmanager" ]; then
     echo "Starting Job Manager"
-    sed -i -e "s/jobmanager.rpc.address: localhost/jobmanager.rpc.address: ${JOB_MANAGER_RPC_ADDRESS}/g" "$FLINK_HOME/conf/flink-conf.yaml"
-    echo "blob.server.port: 6124" >> "$FLINK_HOME/conf/flink-conf.yaml"
-    echo "query.server.port: 6125" >> "$FLINK_HOME/conf/flink-conf.yaml"
+	
+	conf_file="$FLINK_HOME/conf/flink-conf.yaml"
+	
+	if grep -E "^jobmanager\.rpc\.address:.*" $(conf_file); then 
+		sed -i -e "s/jobmanager.rpc.address:*/jobmanager.rpc.address: ${JOB_MANAGER_RPC_ADDRESS}/g" $(conf_file)
```

Comment:
Not sure how the `$(conf_file)` syntax actually works here—would this not try to execute the filename as a command? I imagine it should be "$conf_file" instead.

Since this variable is used elsewhere, could you elevate it to a global `FLINK_CONF` underneath `JOB_MANAGER_RPC_ADDRESS`?

Reply:
I forced some changes. check.